src/test-run/index.js

```diff
@@ -83,10 +103,12 @@ export default class TestRun extends Session {
         // TODO
     }
 
-    executeCommand (command) {
+    executeCommand (command, callsite) {
         if (this.pendingCommand)
             throw new Error('An attempt to execute a command when a previous command is still being executed was detected.');
 
+        this.currentCommandCallsite = callsite;
```

Comment:
Oh, `pendingCommand` may not exist. Then we shouldn't store callsite in `pendingCommand` I guess.